script/cmake-build

```diff
@@ -67,7 +67,7 @@ OT_CMAKE_NINJA_TARGET=${OT_CMAKE_NINJA_TARGET-}
 OT_SRCDIR="$(cd "$(dirname "$0")"/.. && pwd)"
 readonly OT_SRCDIR
 
-OT_PLATFORMS=(simulation posix android-ndk)
+OT_PLATFORMS=(simulation posix android-ndk toranj)
```

Comment:
Adding `toranj` in `OT_PLATFORMS` can confusing.
`toranj` is not a new platform., it is a test  framework which uses the simulation platform.